tests/testsuite/resolve.rs

```diff
@@ -354,6 +354,12 @@ fn test_resolving_minimum_version_with_transitive_deps() {
         pkg!("bar" => [dep_req("util", ">=1.0.1")]),
     ]);
 
+    if !nightly_features_allowed() {
```

Comment:
I feel like I've seen other cargo tests use something like `is_nightly` already, perhaps grep for `#![feature`?

Reply:
good call switched to `is_nightly`